Gemfile

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'cocoapods', '~> 1.5'
```

Comment:
I added CocoaPods as a Bundler dependency to ensure everyone uses the same version of it.

Reply:
I'd prefer to not specify all this here. The less Ruby crap in the repo, the better. (And none of the other projects I maintain have a Gemfile.)